wavelength-plugins/exp.js

```diff
@@ -108,6 +108,18 @@ function addExp(user, room, amount) {
 					Monitor.adminlog(user.userid + ' has earned a chatroom for reaching level ' + level + '!');
 					reward = 'a Chatroom. To claim your chatroom, Contact a Leader (&) or Administrator (~).';
 					break;
+				case 45:
```

Comment:
45 and 50 feels too high for this tbh

Reply:
tbh I was just setting it in the ongoing increments until HoeenHero changes it

Reply:
Id agree, custom symbol is stupid so maybe just allow both of these for level 5?